source/extensions/filters/udp/dns_filter/dns_parser.h

```diff
@@ -0,0 +1,189 @@
+#pragma once
```

Comment:
This file still needs a fuzzer. Can you add a TODO about this? This will be required before we take this filter out of alpha. cc @asraa

Reply:
Actively working on this now.  Added a note in dns_parser.cc